### PR TITLE
Localize short names for black and white

### PR DESCRIPTION
**Pull request by lykahb**
This replaces the B&W in the graph and the stats box. I took a liberty to update the other languages. Luckily, all supported languages have the colors begin with the different letters.

image

### CODE REVIEW

File: katrain/gui.kv [hunk 1]
```diff
@@ -499,7 +499,7 @@
     StatsLabel:
         id: pointloss_label
         label: i18n._('stats:pointslost') if root.points_lost is None or root.points_lost > 0 else i18n._('stats:pointsgained')
-        text: '{}: {:.1f}'.format(root.player,abs(root.points_lost)) if root.points_lost is not None else '...'
+        text: '{}: {:.1f}'.format(i18n._('short color ' + root.player), abs(root.points_lost)) if root.points_lost is not None else '...'
```
Comment (lykahb) [inline katrain/gui.kv:502]: That's more logic than most of the UI elements have. I considered extracting this into Python and passing `points_delta_label` & `points_delta` but decided to keep the scope more focused.

File: katrain/i18n/locales/ua/LC_MESSAGES/katrain.po [hunk 2]
```diff
@@ -185,8 +178,8 @@ msgstr "Визначити область для аналізу"
 #. Status message
 msgid "analysis:region:start"
 msgstr ""
-"Потягніть для вибору області інтересу. Виберіть одну точку для"
-"скасування області."
+"Потягніть для вибору області інтересу. Виберіть одну точку для скасування "
+"області."
```
Comment (lykahb) [inline katrain/i18n/locales/ua/LC_MESSAGES/katrain.po:182]: In the past PR #712 I formatted this file manually rather than used the i18n.py script. So there would be many unrelated changes in this file.

File: katrain/i18n/locales/ua/LC_MESSAGES/katrain.po [hunk 3]
```diff
@@ -287,6 +280,12 @@ msgstr "Графік очок"
 msgid "Jigo"
 msgstr "Джиґо"
 
+msgid "short color B"
+msgstr "Ч"
+
+msgid "short color W"
+msgstr "Б"
```
Comment (lykahb) [inline katrain/i18n/locales/ua/LC_MESSAGES/katrain.po:287]: The new messages for the graph and stats.

File: katrain/i18n/locales/en/LC_MESSAGES/katrain.po [hunk 4]
```diff
@@ -765,8 +771,8 @@ msgstr "KataGo"
 
 msgid "aihelp:default"
 msgstr ""
-"Full strength KataGo AI. Strength is affected by 'max visits' and 'model' "
-"in the general settings 'engine' section and engine configuration file. No "
```
Comment (lykahb) [inline katrain/i18n/locales/en/LC_MESSAGES/katrain.po:769]: The i18n.py reformatted it.

File: katrain/gui.kv [hunk 5]
```diff
@@ -499,7 +499,7 @@
     StatsLabel:
         id: pointloss_label
         label: i18n._('stats:pointslost') if root.points_lost is None or root.points_lost > 0 else i18n._('stats:pointsgained')
-        text: '{}: {:.1f}'.format(root.player,abs(root.points_lost)) if root.points_lost is not None else '...'
+        text: '{}: {:.1f}'.format(i18n._(f'short color {root.player}'), abs(root.points_lost)) if root.points_lost is not None else '...'
```
Comment (lykahb) [inline katrain/gui.kv:502]: That's more logic than most of the UI elements have. I considered extracting this into Python and passing points_delta_label & points_delta but decided to keep the scope more focused.

File: README.md [hunk 6]
```diff
@@ -39,10 +39,10 @@ KaTrain is a tool for analyzing games and playing go with AI feedback from KataG
 <a href="http://github.com/sanderland/katrain/blob/master/README.md"><img alt="English" src="https://github.com/sanderland/katrain/blob/master/katrain/img/flags/flag-uk.png" width=50></a>
 <a href="http://translate.google.com/translate?sl=en&tl=de&u=https%3A%2F%2Fgithub.com%2Fsanderland%2Fkatrain%2Fblob%2Fmaster%2FREADME.md"><img alt="German" src="https://github.com/sanderland/katrain/blob/master/katrain/img/flags/flag-de.png" width=50></a>
 <a href="http://translate.google.com/translate?sl=en&tl=fr&u=https%3A%2F%2Fgithub.com%2Fsanderland%2Fkatrain%2Fblob%2Fmaster%2FREADME.md"><img alt="French" src="https://github.com/sanderland/katrain/blob/master/katrain/img/flags/flag-fr.png" width=50></a>
+<a href="http://translate.google.com/translate?sl=en&tl=uk&u=https%3A%2F%2Fgithub.com%2Fsanderland%2Fkatrain%2Fblob%2Fmaster%2FREADME.md"><img alt="Ukrainian" src="https://github.com/sanderland/katrain/blob/master/katrain/img/flags/flag-ua.png" width=50></a>
```
Comment (lykahb) [inline README.md:42]: I missed the list of localizations in the previous PR. Added it in the same order as in the UI.

**Comment by sanderland** [comment]
Thanks. Will pass this by the community for any local conventions.

**Review by sanderland — APPROVED** [review]

**Comment by kaorahi** [comment]
Is this a wrong way to show the localized color names in the "Player Setup" menu?

localized

```
--- a/katrain/gui.kv
+++ b/katrain/gui.kv
@@ -200,6 +200,7 @@
         pos: (image.x, image.y)
         size: image.size
         font_size: root.min_size * 1.2 / (1 + 1 * len(self.text))
+        font_name: i18n.font_name
 
 # UI components
 
@@ -934,7 +935,7 @@
         ClickableCircle:
             size_hint: 0.8, 0.8
             pos_hint: {'center_x':0.5,'center_y':0.5}
-            text: root.player[0]
+            text: i18n._(f'short color {root.player}')
             player: root.player
             on_left_press: root.parent.swap_players()
     MDBoxLayout:
```
